src/Components/SignIn.js

```diff
@@ -13,20 +14,63 @@ const validation = Yup.object().shape({
     .required('Must enter a password')
 })
 
+const SIGN_IN = gql`
+  mutation sign_in($data: SigninInput!){
+    signin(data: $data){
+      user{
+        id
+      }
+      jwt
+    }
+  }
+`
+
 export default function SignIn () {
+  const [error, setError] = useState(null)
+
+  const [signin] = useMutation(SIGN_IN, {
+    onCompleted: LoginSuccess,
+    onError: LoginFailed
+  }
+  )
```

Comment:
Close the parenthesis at the same level of the curly bracket:
```suggestion
  const [signin] = useMutation(SIGN_IN, {
    onCompleted: LoginSuccess,
    onError: LoginFailed
  })
```